spec/support/factory_bot.rb

```diff
@@ -5,6 +5,7 @@
     sequence(:email) {|n| "user#{n}@example.com" }
     password { 'password' }
     password_confirmation { 'password' }
+    username { 'testusername' }
```

Comment:
Since `username` seems to be unique,

```ruby
validates :username, presence: true, uniqueness: true
```

we will also want to make sure this can handle creating many users in the same test.  Line 8 can be updated and modeled of how line 5 ensures a test email will still be unique